raiden/storage/serialization/types.py

```diff
@@ -184,17 +184,17 @@ def message_event_schema_deserialization(
         Signature: BytesField,
         TransactionHash: BytesField,
         # Ints
-        BlockExpiration: fields.Int,
-        BlockNumber: fields.Int,
-        FeeAmount: fields.Int,
-        LockedAmount: fields.Int,
-        BlockGasLimit: fields.Int,
-        MessageID: fields.Int,
-        Nonce: fields.Int,
-        PaymentAmount: fields.Int,
-        PaymentID: fields.Int,
-        PaymentWithFeeAmount: fields.Int,
-        TransferID: fields.Int,
+        BlockExpiration: IntegerToStringField,
+        BlockNumber: IntegerToStringField,
+        FeeAmount: IntegerToStringField,
+        LockedAmount: IntegerToStringField,
+        BlockGasLimit: IntegerToStringField,
+        MessageID: IntegerToStringField,
```

Comment:
here a `Int` field should be enough. Not sure if this would increase usability in migrations.

Reply:
Should we keep this for consistency (all integers are converted to strings during serialization)?

Reply:
Fine for me